docs/en/operation/Performance-Tuning.md

```diff
@@ -246,6 +259,17 @@ Alluxio workers matches the the number of HDFS datanodes, set
 `alluxio.worker.ufs.instream.cache.max.size=<value of HDFS setting dfs.datanode.handler.count>`
 under the assumption that the workload is spread evenly over Alluxio workers.
 
+## Job Service Tuning
+
+### Job Service Capacity
+Job service limits the total number of currently running jobs to control its resource usage. 
+If jobs tend to be created in large batches, consider increasing `alluxio.job.master.job.capacity` to a larger value than the default 100K. 
```

Comment:
Do you think it makes sense to mention also the internal capacity for a distributed command https://github.com/Alluxio/alluxio/blob/2f7c61c7cf8cf38cdb0987902e8ffe4bc2d02a2f/shell/src/main/java/alluxio/cli/fs/command/AbstractDistributedJobCommand.java#L33
and also this is possible overriden by an input argument?

Also what happens when the capacity is full? Will new jobs be rejected or blocking?

Reply:
elaborate what a "job" means here? for example, a single distributedLoad operation can have X jobs controlling the % of the total job server capacity consumed by the single distributedLoad operation.